service/api.go

```diff
@@ -667,14 +667,15 @@ func (as *APIServer) validateProjectConfig(w http.ResponseWriter, r *http.Reques
 		errs = append(errs, validationErr)
 	}
 
+	errs = append(errs, validator.CheckProjectErrors(project, input.IncludeLong)...)
+	if len(errs) > 0 && input.Quiet {
```

Comment:
Even if len(errs) == 0, f input is quiet we want to return here, right?

Reply:
Oh correct